main/pom.xml

```diff
@@ -60,9 +60,8 @@
             <artifactId>apache-mime4j-dom</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-core</artifactId>
-            <scope>runtime</scope>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
```

Comment:
Do we need SLF4J at all or we can remove it too?

Reply:
Let remove it then.